File: Expenses/Expenses.Model/Dto/AddCashFlowDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Expenses.Model.Dto
{
    public class AddCashFlowDto
    {
        public DateTime DateTime { get; set; }
        public Guid CategoryGuid { get; set; }
        public Money Amount { get; set; }
        public List<CashFlowDetail> Details { get; set; }
        public Guid WalletGuid { get; set; }
    }
}

File: Expenses.AndroidClient/Expenses.Repository/src/main/java/net/azurewebsites/expenses_by_klaudia/model/AddCashFlowDto.java
package net.azurewebsites.expenses_by_klaudia.model;

import java.util.Date;
import java.util.List;
import java.util.UUID;

public class AddCashFlowDto {
    public Date DateTime;
    public UUID CategoryGuid;
    public Money Amount;
    public List<CashFlowDetail> Details;
    public UUID WalletGuid;
}
